manifests/2.19.1/opensearch-2.19.1.yml

```diff
@@ -71,6 +71,8 @@ components:
       - windows
     depends_on:
       - common-utils
+      - job-scheduler
```

Comment:
I think 2.19.1 was already released and i think the version bump to 2.19.2 was blocked. Do we want to update it here as it should already be fixed?

Reply:
Yes because the 2.19.2 autobump PR in build havent merged yet and it is based on 2.19.1.

Thanks.